### PR TITLE
biogenerator can print flour sacks and enzyme now, also some boxes idk

### PR DESCRIPTION
**Pull request by Hatterhat**
## About The Pull Request
port of tgstation/tgstation#40394
todo: look at porting tgstation/tgstation#44744 and adjusting cit cardboard boxes for it
## Why It's Good For The Game
sometimes you just don't have flour i guess i dunno
## Changelog
:cl: FantasticFwoosh, ported by Hatterhat
add: Both reagent universal enzyme & sacks of flour are now available for their respective costs from the biogenerator.
add: New crafting recipies for donut boxes, eggboxes & candle boxes have been added to cardboard recipes for the collective benefit of service personnel and the station.
/:cl:

### CODE REVIEW

**Comment by CameronWoof** [comment]
I don't really like this. Botany can already produce both of these products by growing wheat or tomatoes (and fermenting them in a barrel to get enzyme). I don't like shifting them towards growing less stuff.

**Comment by davidchan13** [comment]
I mean, they could already grow meat wheat before synthmeat slabs got added to the biogenerator. 

I'm indifferent on the flour sacks but +1 to the enzyme as tg and other servers already allow it to be made, and surprisingly few botanists know they can produce it.

**Comment by Owai-Seek** [comment]
I do and dont mind this addition as it makes things a tad easier as a non-botanist. The process of growing and creating enzyme/flour is just extra tedium that a chef shouldn't have to deal with. At the same time, it makes cargo packs for the chef even less relevant.

This wouldn't be as much of a problem if chefs had their own growing area on citadel, but they currently don't, so getting almost anything from botany can be like pulling teeth. That doesn't even take into consideration that a new botanist can come in, walk into botany, and start shoving earthsblood/weaponized bullshit into the fridge by mistake.

I agree with the change, Botany still has several items it "needs" to make. IE: Tower Caps (For Wood), Soybeans (For Pastry Dough), Ambrosia Gaia (To make botany not suck ass), and bounty items.

File: code/game/objects/items/stacks/sheets/sheet_types.dm [hunk 1]
```diff
@@ -419,17 +419,21 @@ GLOBAL_LIST_INIT(cardboard_recipes, list ( \
 	null, \
 	new/datum/stack_recipe("cardborg suit", /obj/item/clothing/suit/cardborg, 3), \
 	new/datum/stack_recipe("cardborg helmet", /obj/item/clothing/head/cardborg), \
+	new/datum/stack_recipe("pizza box", /obj/item/pizzabox), \
```
Comment (Trilbyspaceclone) [inline code/game/objects/items/stacks/sheets/sheet_types.dm]: Alread on line 406

Comment (Hatterhat) [reply code/game/objects/items/stacks/sheets/sheet_types.dm]: why the fuck is it all the way up there

**Review by Ghommie — APPROVED** [review]
Feel free to re-add the challenge of making enzyme/flour later. I want this merged because of the sweet cardboard recipes change.

**Pull request merged**